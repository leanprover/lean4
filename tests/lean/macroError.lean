macro "foo" : term => Lean.Macro.throwError "Test Error"

#check foo
