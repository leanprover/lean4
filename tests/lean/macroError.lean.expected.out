macroError.lean:3:7-3:10: error: Test Error
